cmd/controller/main.go

```diff
@@ -43,6 +47,27 @@ func main() {
 	lo.Must0(op.AddHealthzCheck("cloud-provider", awsCloudProvider.LivenessProbe))
 	cloudProvider := metrics.Decorate(awsCloudProvider)
 
+	client := op.Manager.GetAPIReader()
+	if client == nil {
```

Comment:
Is this possible?

Reply:
After reading through the source from controller-runtime the answer appears to be no, we can remove this check.